src/test/java/uk/gov/justice/probation/courtcaseservice/controller/CourtCaseControllerIntTest.java

```diff
@@ -140,13 +143,25 @@ public void shouldGetCaseWhenExists() {
                 .body("previouslyKnownTerminationDate", equalTo(LocalDate.of(2010, 1, 1).format(DateTimeFormatter.ISO_LOCAL_DATE)))
                 .body("suspendedSentenceOrder", equalTo(true))
                 .body("breach", equalTo(true))
+                .body("crn", equalTo("X320741"))
+                .body("cro", equalTo("311462/13E"))
+                .body("pnc", equalTo("A/1234560BA"))
+                .body("listNo", equalTo("3rd"))
+                .body("courtCode", equalTo(COURT_CODE))
+                .body("sessionStartTime", equalTo(startTime))
+                .body("lastUpdated", equalTo(startTime))
                 .body("defendantName", equalTo(DEFENDANT_NAME))
                 .body("defendantAddress.line1", equalTo("27"))
                 .body("defendantAddress.line2", equalTo("Elm Place"))
                 .body("defendantAddress.postcode", equalTo("ad21 5dr"))
                 .body("defendantAddress.line3", equalTo("Bangor"))
-                .body("defendantAddress.line4", equalTo(null))
-                .body("defendantAddress.line5", equalTo(null));
+                .body("defendantAddress.line4", nullValue())
+                .body("defendantAddress.line5", equalTo(null))
```

Comment:
Would ideally be consistently using `nullValue()` or `equalTo(null)` but it really doesn't matter

Reply:
Sorry - I did that to see alternative and meant to restore equalTo(null). Oversight.

Reply:
Put it back to equalTo(null)